### PR TITLE
Design spike into what the service will look like when paused

### PR DESCRIPTION
**Pull request by ralph-hawkins**

### CODE REVIEW

File: app/views/school/home/show.html.erb [hunk 1]
```diff
@@ -15,69 +15,64 @@
...
+            In the 2020 to 2021 academic year, your <%= @school.institution_type %> recieved:
+          </p>
+          <ul class="govuk-list govuk-list--bullet">
+            <li><%= @school.std_device_allocation&.devices_ordered %> laptops and tablets</li>
+            <li><%= @school.coms_device_allocation&.devices_ordered %> routers</li>
+            <li>extra mobile data for 10 accounts</li>
+          </ul>
+        <% elsif @school.std_device_allocation&.devices_ordered.to_i.positive? && @school.coms_device_allocation&.devices_ordered.to_i == 0 %>
+            <p class="govuk-body">
+              In the 2020 to 2021 academic year, your <%= @school.institution_type %> recieved <%= @school.std_device_allocation&.devices_ordered %> laptops and tablets.
```
Comment (nigel-lowry) [inline app/views/school/home/show.html.erb:59]: Typo

File: app/views/school/home/show.html.erb [hunk 2]
```diff
@@ -15,69 +15,64 @@
...
+            </p>
+        <% elsif @school.std_device_allocation&.devices_ordered.to_i == 0 && @school.coms_device_allocation&.devices_ordered.positive %>
+          <p class="govuk-body">
+              In the 2020 to 2021 academic year, your <%= @school.institution_type %> recieved <%= @school.std_device_allocation&.devices_ordered %> routers.
```
Comment (nigel-lowry) [inline app/views/school/home/show.html.erb:63]: Typo

File: app/views/shared/_invite_someone_who_can.html.erb [hunk 3]
```diff
@@ -1,19 +1,10 @@
...
-  <li>understands what devices are needed and who will get them</li>
-  <li>can configure devices or give technical information</li>
-</ul>
-
-<h2 class="govuk-heading-s">
-  Get consent for us to share their information
-</h2>
 <p class="govuk-body">
-  When you invite someone, their name, email address and <span class="app-no-wrap">phone number</span> will be shared with TechSource. Before you invite them:
+  When you invite someone, their name, email address and <span class="app-no-wrap">phone number</span> will be shared with Computacenter (UK) Limited, the company that provides the Support Portal. Before you invite them:
```
Comment (nigel-lowry) [inline app/views/shared/_invite_someone_who_can.html.erb:3]: This looks like a permanent change

File: app/views/responsible_body/home/show.html.erb [hunk 4]
```diff
@@ -16,38 +16,43 @@
...
+            <p class="govuk-body">
+              In the 2020 to 2021 academic year, your <%= @responsible_body.humanized_type %> recieved <%= @responsible_body.std_device_allocation&.devices_ordered %> routers.
+            </p>
+          <% else %>
+            <p class="govuk-body">
+              Your <%= @school.institution_type %> did not order any laptops, tablets or routers in the 2020 to 2021 academic year.
+            </p>
+          <% end %>
+        <% else %>
+          <!-- Can we show sum of all the laptops and tablets, and routers for the LA? -->
```
Comment (nigel-lowry) [inline app/views/responsible_body/home/show.html.erb:46]: Is this something we want to do?

File: app/views/school/home/show.html.erb [hunk 5]
```diff
@@ -15,69 +15,64 @@
...
-          <li>read more about the scheme</li>
-        </ul>
+    <div class="app-card">
+      <h2 class="govuk-heading-m">
+        Ordering is now closed for devices and internet access
+      </h2>
+      <% if @school.in_virtual_cap_pool? %>
+        // IF TRUST ORDERED LAPTOPS AND ROUTERS
+          <p class="govuk-body">
+            In the 2020 to 2021 academic year, <%= TRUST %> recieved:
```
Comment (nigel-lowry) [inline app/views/school/home/show.html.erb:25]: Typo

File: app/views/shared/_invite_someone_who_can.html.erb (hunk 3 above)
Comment (ralph-hawkins) [reply app/views/shared/_invite_someone_who_can.html.erb:3]: True, I'll make this change in a seperate branch

Comment (ralph-hawkins) [reply app/views/shared/_invite_someone_who_can.html.erb:3]: True, I'll make this change separately

**Comment by markhallen** [comment]
[Dev epic for Closing down the Service for June 30th](https://trello.com/c/aZoPVHii/2142-dev-epic-for-closing-down-the-service-for-june-30th)

**Pull request closed without merging**